Neos.ContentRepository/Classes/Command/NodeCommandController.php

```diff
@@ -111,11 +116,57 @@ public function repairCommand($nodeType = null, $workspace = 'live', $dryRun = f
             $plugin = $pluginConfiguration['object'];
             $this->outputLine('<b>' . $plugin->getSubCommandShortDescription('repair') . '</b>');
             $this->outputLine();
+            if ($plugin instanceof EventDispatchingNodeCommandControllerPluginInterface) {
+                $this->attachPluginEventHandlers($plugin, $dryRun);
+            }
+            /** @noinspection PhpMethodParametersCountMismatchInspection */
             $plugin->invokeSubCommand('repair', $this->output, $nodeType, $workspace, $dryRun, $cleanup, $skip, $only);
             $this->outputLine();
         }
 
-        $this->outputLine('Node repair finished.');
+        if ($dryRun) {
+            $this->outputLine('Node repair dry run finished.');
+        } else {
+            $this->outputLine('<success>Node repair finished.</success>');
+        }
+    }
+
+    /**
+     * @param EventDispatchingNodeCommandControllerPluginInterface $plugin
+     * @param bool $dryRun
+     * @return void
+     */
+    private function attachPluginEventHandlers(EventDispatchingNodeCommandControllerPluginInterface $plugin, bool $dryRun)
```

Comment:
```suggestion
    private function attachPluginEventHandlers(EventDispatchingNodeCommandControllerPluginInterface $plugin, bool $dryRun): void
```

Reply:
That's not a new method. I didn't want to adjust the signature of existing ones because they might be overridden in extended implementations